appinfok8s.go

```diff
@@ -120,3 +124,16 @@ func (ai AppengineInfoK8s) NumInstances(moduleName, version string) (int, error)
 		return instances, nil
 	}
 }
+
+func (ai AppengineInfoK8s) GcpServiceAccountName(saName string) (string, error) {
+	namespace := ai.NativeProjectID()
```

Comment:
Should this be Data project?

Reply:
Possibly? I used the same param as the other functions here `NumInstances` `ModuleHasTraffic` are both also using the `NativeProjectID()`

Reply:
In retrospect, i think we do in fact want Native here. In a DR scenario we would have the GOOGLE_PROJECT set to the dr project but picking up data from say pendo-test or where ever else and the app wouldn't be running in the pendo-test namespace

Reply:
Yeah that's what we did in the DR test so ok 👍